src/pykeen/nn/emb.py

```diff
@@ -140,6 +140,11 @@ def embedding_dim(self) -> int:
         warnings.warn("The embedding_dim property is deprecated. Use .shape instead.", DeprecationWarning)
         return int(np.prod(self.shape))
 
+    @property
+    def device(self) -> torch.device:
+        """Return the device."""
+        return get_preferred_device(module=self, allow_ambiguity=True)
```

Comment:
does making `allow_ambiguity=False` more strict create problems during normal runtime?

Reply:
It would prevent `mod.to(mod.device)` to work, if the module has mixed devices.